src/components/structures/login/Registration.js

```diff
@@ -168,26 +169,19 @@ module.exports = React.createClass({
             busy: true,
             doingUIAuth: false,
         });
-        this.props.onLoggedIn({
-            userId: response.user_id,
-            deviceId: response.device_id,
-            homeserverUrl: this.state.hsUrl,
-            identityServerUrl: this.state.isUrl,
-            accessToken: response.access_token,
-        });
```

Comment:
This was the same thing as was done after the trackPromise: we should just do it then.